src/site/markdown/server/manage-webadmin.md

```diff
@@ -1,2379 +1,2409 @@
-
-```
-Location: /tasks/3294a976-ce63-491e-bd52-1b6f465ed7a2
-```
-
-Response codes:
-
- - 201: Success. Corresponding task id is returned.
- - 400: Error in the request. Details can be found in the reported error.
-
-The scheduled task will have the following type `MessageIdReIndexingTask` and the following `additionalInformation`:
-
-```
-{
-  "messageId":"18"
-}
-```
-
-Warning: During the re-indexing, the result of search operations might be altered.
-
-## Task management
-
-Some webadmin features schedules tasks. The task management API allow to monitor and manage the execution of the following tasks.
-
-Note that the `taskId` used in the following APIs is returned by other WebAdmin APIs scheduling tasks.
-
- - [Getting a task details](#Getting_a_task_details)
- - [Awaiting a task](#Awaiting_a_task)
- - [Cancelling a task](#Cancelling_a_task)
- - [Listing tasks](#Listing_tasks)
-
-### Getting a task details
-
-```
-curl -XGET http://ip:port/tasks/3294a976-ce63-491e-bd52-1b6f465ed7a2
-```
-
-An Execution Report will be returned:
-
-```
-{
-    "submitDate": "2017-12-27T15:15:24.805+0700",
-    "startedDate": "2017-12-27T15:15:24.809+0700",
-    "completedDate": "2017-12-27T15:15:24.815+0700",
-    "cancelledDate": null,
-    "failedDate": null,
-    "taskId": "3294a976-ce63-491e-bd52-1b6f465ed7a2",
-    "additionalInformation": {},
-    "status": "completed",
-    "type": "typeOfTheTask"
-}
-```
-
-Note that:
-
- - `status` can have the value:
-    - `waiting`: The task is scheduled but its execution did not start yet
-    - `inProgress`: The task is currently executed
-    - `cancelled`: The task had been cancelled
-    - `completed`: The task execution is finished, and this execution is a success
-    - `failed`: The task execution is finished, and this execution is a failure
-
- - `additionalInformation` is a task specific object giving additional information and context about that task. The structure
-   of this `additionalInformation` field is provided along the specific task submission endpoint.
-
-Response codes:
-
- - 200: The specific task was found and the execution report exposed above is returned
- - 400: Invalid task ID
- - 404: Task ID was not found
-
-### Awaiting a task
-
-One can await the end of a task, then receive it's final execution report.
-
-That feature is especially usefull for testing purpose but still can serve real-life scenari.
-
-```
-curl -XGET http://ip:port/tasks/3294a976-ce63-491e-bd52-1b6f465ed7a2/await
-```
-
-An Execution Report will be returned.
-
-Response codes:
-
- - 200: The specific task was found and the execution report exposed above is returned
- - 400: Invalid task ID
- - 404: Task ID was not found
-
-### Cancelling a task
-
-You can cancel a task by calling:
-
-```
-curl -XDELETE http://ip:port/tasks/3294a976-ce63-491e-bd52-1b6f465ed7a2
-```
-
-Response codes:
- - 204: Task had been cancelled
- - 400: Invalid task ID
-
-### Listing tasks
-
-A list of all tasks can be retrieve
```

Comment:
`The component is referenced by its URL encoded name`

Reply:
fixed in mschnitzler/james-project@d7d8e57
Also undid the line delimiter change that I introduced (as [mentioned](https://github.com/linagora/james-project/pull/1856#pullrequestreview-169170842) by @trantienduchn).